instapi/models/user.py

```diff
@@ -88,6 +88,18 @@ def media_count(self) -> int:
         """
         return cast(int, self.user_detail()["media_count"])
 
+    @property
+    def get_phone_number(self) -> str:
+        """
+        Return user's phone number
+
+        :return: str
+        """
+        detail = self.user_detail()
+        phone_number = detail["public_phone_country_code"] + detail["public_phone_number"]
+
+        return "+{}".format(phone_number) if phone_number else ""
```

Comment:
Personally, I don't like the idea of returning an empty string when phone number is not present, I would rather return `None` to show that the phone number is missed.